src/main/resources/log4j2.xml

```diff
@@ -15,38 +15,62 @@
 ~ See the license for the specific language governing permissions and
 ~ limitations under the license.
 -->
-<Configuration status="ERROR">
+<Configuration status="DEBUG">
     <Properties>
         <Property name="DEFAULT_HOME">$${env:JENKINS_HOME:-$${sys:JENKINS_HOME:-.}}</Property>
         <Property name="FALLBACK_LOG_DESTINATION">${DEFAULT_HOME}/logs/audit.log</Property>
         <Property name="AUDIT_LOG_DESTINATION">$${sys:auditFileName:-${FALLBACK_LOG_DESTINATION}}</Property>
         <Property name="IANA_ENTERPRISE_NUMBER">$${sys:enterpriseNumber:-.}</Property>
-        <Property name="APPENDER_LAYOUT_TYPE">$${sys:appenderType:-.}</Property>
+        <Property name="APPENDER_LAYOUT_TYPE">$${sys:appenderType:-jsonLayout}</Property>
+        <Property name="PORT">1854</Property>
+        <Property name="HOST">localhost</Property>
     </Properties>
+
     <Appenders>
         <Routing name="Routing">
-            <Routes pattern="${APPENDER_LAYOUT_TYPE}">
-                <Route>
-                    <RollingRandomAccessFile name="audit" fileName="${AUDIT_LOG_DESTINATION}" filePattern="${AUDIT_LOG_DESTINATION}.%d{yyyyMMdd_HHmmss}-%i.log">
+            <Script name="RoutingInit" language="Groovy"><![CDATA[
+            (System.properties['appenderType'] == "syslogLayout") ? "syslogLayout" : "jsonLayout";]]>
```

Comment:
Ok, so based on this script, you should likely be able to just use the pattern feature instead like `<Routes pattern="$${sys:appenderType:-jsonLayout}">` (no need to create a Property for it first like you did here; that could potentially be causing issues). That way we don't need to reconfigure log4j, either, when we update the configuration (a small enhancement we can do elsewhere after this).

Reply:
Yes, Thanks. This enhancement will be helpful not to reconfigure all the time. I was able to understand that after changing the configuration status to `debug`. A lot of things happens because of reconfiguration.

But with this case, `<Routes pattern="$${sys:appenderType:-jsonLayout}">` is also not working. I have tested it. Maybe I am missing something else minor here!